integration_tests/test_runner/src/utils.rs

```diff
@@ -786,94 +779,6 @@ pub async fn wait_for_balance(
     panic!("User did not attain >= expected balance");
 }
 
-// TODO: launch into deep space
```

Comment:
These functions with // TODO: launch into deep space get upstreamed in the final commit of this PR